pkg/db/query.go

```diff
@@ -131,7 +131,7 @@ type Select struct {
 
 func (s *Select) WhereRaw(stmt string, args ...interface{}) *Select {
 	s.addWhere()
-	s.buffer.WriteString(fmt.Sprintf(" (%s)", stmt))
+	s.buffer.WriteString(fmt.Sprintf(" %s", stmt))
```

Comment:
Are we ok with this change? I've changed it because it caused failing [this](https://github.com/kyma-incubator/reconciler/pull/487/files#diff-36c829107c81e04951311e0d023fe69e017531e811067f4c18cbf3e4cc21e8a9R22) validation